MonoGame.Framework/Color.cs

```diff
@@ -16,6 +16,20 @@ namespace Microsoft.Xna.Framework
     [DebuggerDisplay("{DebugDisplayString,nq}")]
     public struct Color : IEquatable<Color>
     {
+        #region Private Fields
+
+        /// <summary>
+        /// ARGB.
+        /// </summary>
+        private uint _packedValue;
+
+        #endregion
+
+        #region Constructors
+
+        /// <summary>
+        /// Private static initializator for colors.
+        /// </summary>

```

Comment:
Lets not start documenting private members.  They don't show up in the documentation and we have plenty of public APIs that need good docs as it is...  lets focus on making them better instead.